docs/provisioner/08-06-upgrading-shoots.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Upgrade shoots
+type: Tutorials
+---
+
+This tutorial shows how to upgrade Gardener Shoot clusters for Kyma Runtimes.
+
+## Steps
+
+> **NOTE:** To access the Runtime Provisioner, forward the port on which the GraphQL server is listening.
+
+To upgrade a Gardener Shoot cluster used to host the Runtime of a given ID, make a call to the Runtime Provisioner with a **tenant** header using a mutation like this:  
+
+```graphql
+mutation { 
+  upgradeShoot(
+    id: "61d1841b-ccb5-44ed-a9ec-45f70cd1b0d3"
+    config: {
+      gardenerConfig: {
+        kubernetesVersion: "1.15.11"
+        volumeSizeGB: 35
+        machineType: "Standard_D2_v3"
+        diskType: "pd-standard"
+        purpose: "testing"
+        autoScalerMin: 2
+        autoScalerMax: 4
+        maxSurge: 4
+        maxUnavailable: 1
+        enableKubernetesVersionAutoUpdate: false
+        enableMachineImageVersionAutoUpdate: false
+        providerSpecificConfig: { 
+          azureConfig: {
+            zones: ["1", "2"]
+          } 
+        }
+      }
+    }
+  ) 
+}
```

Comment:
I think that the mutation is incorrect as we do not query any fields and I do not think it is allowed.

Reply:
I was told the `gardenerConfig` fields are optional here. Will check, thanks.

Reply:
Maybe this would solve the problem?
```suggestion
mutation { 
  upgradeShoot(
    id: "61d1841b-ccb5-44ed-a9ec-45f70cd1b0d3"
    config: {
      gardenerConfig: {
        kubernetesVersion: "1.15.11"
        volumeSizeGB: 35
        machineType: "Standard_D2_v3"
        diskType: "pd-standard"
        purpose: "testing"
        autoScalerMin: 2
        autoScalerMax: 4
        maxSurge: 4
        maxUnavailable: 1
        enableKubernetesVersionAutoUpdate: false
        enableMachineImageVersionAutoUpdate: false
        providerSpecificConfig: { 
          azureConfig: {
            zones: ["1", "2"]
          } 
        }
      }
    }
  ){
    id operation state message
  } 
}
```

Reply:
Yes, that is exactly what I meant

Reply:
Thanks, guys! Fixed that :)